testing/src/main/java/keywhiz/testing/JsonHelpers.java

```diff
@@ -56,8 +57,12 @@ private JsonHelpers() { /* singleton */ }
    * @return {@code object} as a JSON string
    * @throws JsonProcessingException if there is an error encoding {@code object}
    */
-  public static String asJson(Object object) throws JsonProcessingException {
-    return MAPPER.writeValueAsString(object);
+  public static String asJson(Object object) {
+    try {
+      return MAPPER.writeValueAsString(object);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
+    }
   }
```

Comment:
Personally I feel that checked exceptions in test helper methods just contribute to API cruft, IDE nagging (wrap with try/catch, etc.), and general obnoxiousness.